doc/.openapi-generator/FILES

```diff
@@ -70,7 +70,6 @@ Models/Workspace.md
 Models/WorkspaceAccessControl.md
 Models/WorkspaceFile.md
 Models/WorkspaceRole.md
-Models/WorkspaceSecret.md
```

Comment:
We should actually remove that file from the repo. (and at some point automate this, files are correctly generated when a new one appears but not removed automatically)